### PR TITLE
feat(Variables): Support for `--param` CLI options

### PR DESCRIPTION
**Pull request by pgrzesik**
Due to the fact that resolution logic for it is implemented in `dashboard-plugin`, actual support will be introduced after merging https://github.com/serverless/dashboard-plugin/pull/670/files

Addresses: #9817

### CODE REVIEW

File: lib/cli/commands-schema/common-options/global.js [hunk 1]
```diff
@@ -5,6 +5,10 @@ module.exports = {
   version: { usage: 'Show version info', shortcut: 'v', type: 'boolean' },
   verbose: { usage: 'Show verbose logs', type: 'boolean' },
   debug: { usage: 'Namespace of debug logs to expose (use "*" to display all)', type: 'string' },
+  param: {
```
Comment (medikoo) [inline lib/cli/commands-schema/common-options/global.js:8]: This is a service specific option, so should rather be defined in `service.js`

Comment (pgrzesik) [reply lib/cli/commands-schema/common-options/global.js]: great point, fixed

**Review by medikoo — APPROVED** [review]
👍

**Comment by pgrzesik** [comment]
@mnapoli If you'll have any suggestions regarding docs, let me know and I will followup with another PR

**Comment by mnapoli** [comment]
👍  just checked, all good for me!

**Comment by pgrzesik** [comment]
thanks @mnapoli 🙇

**Pull request merged**